counterpartylib/protocol_changes.json

```diff
@@ -128,8 +128,15 @@
     "enhanced_sends": {
         "minimum_version_major": 9,
         "minimum_version_minor": 55,
-        "minimum_version_revision": 4,
+        "minimum_version_revision": 3,
         "block_index": 999999,
         "testnet_block_index": 0
```

Comment:
Please change the testnet_block_index to `1179400` for both `short_tx_type_id` and `enhanced_sends` in order to help Circle tests pass.